.github/workflows/ci.yml

```diff
@@ -31,6 +31,7 @@ env:
   # the Docker daemon only downloads 3 layers concurrently which prevents the other pull from making any progress.
   # This value should be greater than the time taken for the longest image pull.
   TESTCONTAINERS_PULL_PAUSE_TIMEOUT: 600
+  TESTCONTAINERS_SKIP_ARCHITECTURE_CHECK: true
```

Comment:
@wendigo WDYT about setting this in the CI for all jobs?

Reply:
I'm fine with that as long as we are not testing on ARM :)

Reply:
There is a risk that one may spend a lot of time troubleshooting issues because of this. However I think in CI it won't work at all and when trying to reproduce the issue they will notice the arch mismatch.